### PR TITLE
docs: Add scoop installer instructions

### PR DESCRIPTION
**Pull request by littleli**
Add installation instruction using scoop installer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +50,23 @@ To upgrade:
...
+
+```console
+scoop bucket add scoop-clojure https://github.com/littleli/scoop-clojure
+scoop install trenchman
+```
+
+To upgrade use:
+
+```console
+scoop update *
```
Comment (athos) [inline README.md:65]: This line is doing more than just updating Trenchman, so I think I'd prefer to just `scoop update trenchman`.

**Review by athos — CHANGES_REQUESTED** [review]
Thanks for the PR! It looks good basically 👍 I added just one comment.

File: README.md (hunk 1 above)
Comment (littleli) [reply README.md:65]: Ok. Makes sense. Let me fix this.

**Review by athos — APPROVED** [review]
Thanks for updating! LGTM 👍

**Pull request merged**